packages/slonik/src/dbFilters.ts

```diff
@@ -64,7 +73,7 @@ const applyFilter = (filter: FilterInput, tableName: string) => {
 
 const applyFiltersToQuery = (
```

Comment:
Why is this method named `applyFiltersToQuery` when the other method is named `applyFilter`. Why not call this one `applyFilters`? Alternatively, name the other method `applyFilterToQuery`. Consistency requires the names to be similar. Simpicity argues in favour of `applyFilter(s)`.